lib/mongo_client.js

```diff
@@ -115,7 +115,6 @@ const validOptions = require('./operations/mongo_client_ops').validOptions;
  */
 function MongoClient(url, options) {
   if (!(this instanceof MongoClient)) return new MongoClient(url, options);
-
```

Comment:
nit: random whitespace change, let's try to keep these out of commits unless there are other useful changes.